app/javascript/components/tooltips/Tooltip.tsx

```diff
@@ -262,6 +247,28 @@ export const Tooltip = ({
     modifiers: [{ name: 'offset', options: { offset: [0, 8] } }],
   })
 
+  // Retrieve the HTML contents from the contentEndpoint
+  const { isLoading, isError, data: htmlContent } = useQuery<string>(
+    id,
+    () => {
+      const controller = new AbortController()
+      const signal = controller.signal
+      return Object.assign(
+        // Create a fetch request for the tooltip content, assign the abort controller to the promise
+        // https://react-query.tanstack.com/docs/guides/query-cancellation#using-fetch
+        fetch(contentEndpoint, {
+          method: 'get',
+          signal,
+        }).then((res) => res.text()),
+        { cancel: () => controller.abort() }
+      )
+    },
+    {
+      // Enable the query to fetch only if it isn't in a hidden or error state
+      enabled: !(showState == 'hidden' || showState == 'error'),
```

Comment:
Personally, I find the following version slightly easier to read:

```suggestion
      enabled: showState != 'hidden' && showState != 'error',
```

But feel free to ignore.

Reply:
This PR does not break the popups @neenjaw

Reply:
DeMorgan's law hard at work! I'll change it, it doesn't matter to me